tests/end-to-end/e2e/playground_test.go

```diff
@@ -9,80 +9,111 @@ import (
 	"io"
 	"log"
 	"net/http"
-	"os"
 	"testing"
 	"time"
 
-	retry "github.com/avast/retry-go"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/avast/retry-go"
 	"github.com/kyma-incubator/compass/components/director/pkg/graphql"
 	"github.com/kyma-incubator/compass/tests/end-to-end/pkg/connector"
 	"github.com/kyma-incubator/compass/tests/end-to-end/pkg/gql"
 	"github.com/kyma-incubator/compass/tests/end-to-end/pkg/idtokenprovider"
 	gcli "github.com/machinebox/graphql"
 	"github.com/stretchr/testify/require"
+	"github.com/vrischmann/envconfig"
 )
 
-func TestDirectorPlaygroundAccess(t *testing.T) {
-	const (
-		DirectorURLformat   = "https://%s.%s/director"
-		OAuth2Subdomain     = "compass-gateway-auth-oauth"
-		JWTSubdomain        = "compass-gateway"
-		ClientCertSubdomain = "compass-gateway-mtls"
-	)
+type playgroundTestConfig struct {
+	Gateway struct {
+		Domain               string `envconfig:"DOMAIN"`
+		JWTSubdomain         string
+		OAuth20Subdomain     string `envconfig:"GATEWAY_OAUTH2_SUBDOMAIN"`
```

Comment:
I think it'd be best to change this field to use the same name as env variable or add `envconfig` tag to other config fields. As it is now it can be a bit confusing why only one of these fields has specified tag.

Reply:
Unfortunately, we have to keep it for `GATEWAY_OAUTH20_SUBDOMAIN`, because envconfig doesn't detect this variable automatically.